### PR TITLE
RFC 3032 says labels 0 and 2 are for IPv4 and IPv6 respectively

### PR DESCRIPTION
**Pull request by dgwynne**
This specifies IPv4 and IPv6 inside an MPLS label using the Explicit
NULL labels, which at least makes building a working packet easier.

This is just a checklist to guide you. You can remove it safely.

**Checklist:**
- [ ] If you are new to Scapy: I have checked https://github.com/secdev/scapy/blob/master/CONTRIBUTING.md (esp. section submitting-pull-requests)
- [ ] I squashed commits belonging together
- [ ] I added unit tests or explained why they are not relevant
- [ ] I executed the regression tests for Python2 and Python3 (using `tox` or, `cd test && ./run_tests_py2, cd test && ./run_tests_py3`)

< brief description what this PR will do, e.g. fixes broken dissection of XXX >

< if required - short explanation why you fixed something in a way that may look more complicated as it actually is >

< if required - outline impacts on other parts of the library >

fixes #< add issue number here if appropriate, else remove this line>

### CODE REVIEW

**Comment by gpotter2** [comment]
So what should label be if MPLS is next ? You also need to update the tests

**Comment by dgwynne** [comment]
According to RFC 4182, Explicit NULL labels are now only meaningful on the bottom of the stack, labels further up the stack can be any value. The exception is label 1 for Router Alerts. It's not supposed to be on the bottom of the stack.

It is worth noting that you can use pretty much any label at any point in the stack. It is up to the endpoints to agree on what payload a particular label encapsulates, the next protocol is generally not encoded in MPLs.

File: scapy/contrib/mpls.py [hunk 1]
```diff
@@ -66,5 +66,7 @@ def guess_payload_class(self, payload):
 bind_layers(UDP, MPLS, dport=6635)
 bind_layers(GRE, MPLS, proto=0x8847)
 bind_layers(MPLS, MPLS, s=0)
+bind_layers(MPLS, IP, label=0) # IPv4 Explicit NULL
```
Comment (p-l-) [inline scapy/contrib/mpls.py]: You need to have two spaces before the inline comments.

**Comment by gpotter2** [comment]
(That should restart the appveyor test).

@guedou BTW it would be great if you could give us access to the appveyor repo.

**Comment by guedou** [comment]
@gpotter2 invite sent!

**Comment by gpotter2** [comment]
@guedou Thanks ! (I received the link but it is broken (appveyor internal error), sent a message to the support)

**Comment by guedou** [comment]
Try to use the second email.

**Comment by gpotter2** [comment]
@guedou I tried both. They answered saying you should invite us as "collaborators" rather than "users", and that they were fixing it to make it more intuitive

**Pull request merged**